drawOnePage.js

```diff
@@ -0,0 +1,252 @@
+function drawPageLogIn () {
+    document.getElementById('temp').remove();
+    const submitLogIn = document.getElementById('submitSignIn');
+    submitLogIn.setAttribute('id','submitLogIn');
```

Comment:
assignment to const is between logical operations. Please separate it with empty lines